.github/actions/publish-to-s3/action.yml

```diff
@@ -0,0 +1,107 @@
+name: Publish to S3
+description: Publishes the specified directory using aws s3 sync
+
+inputs:
+  AWS_REGION:
+    description: The region of your AWS bucket
+    default: ca-central-1
+    required: false
+  AWS_ROLE:
+    description: The AWS role to assume. This role must have write permission for your bucket
+    required: true
+  BUCKET_PATH:
+    description: The full path of your bucket, including subdirectories, to which to publish
+    required: true
+  BUILD_DIRECTORY:
+    description: The directory within your repo to publish to S3 (e.g. "./build/")
+    required: true
+  CACHE:
+    description: An optional comma-separated list of all file extensions you wish to have cached for 1 year (e.g. "js,css")
+    default: ""
+    required: false
+  CACHE_DEFAULT:
+    description: An optional default caching policy to apply to all files (e.g. "--cache-control 'max-age=120'")
+    default: ""
+    required: false
+  COMPRESS:
+    description: Whether to compress eligible files (if "true", will compress all of html/css/js/json/svg/xml)
+    default: false
+    required: false
```

Comment:
I can't think of a reason we wouldn't want to compress... I think I'd personally not even make this an option.

Reply:
Okay, I like having it as an option but I'm okay to remove it 👍.

Reply:
Removed.